lib/deprecation_toolkit/minitest_hook.rb

```diff
@@ -13,6 +13,23 @@ def trigger_deprecation_toolkit_behavior
     ensure
       Collector.reset!
     end
+
+    class Plugin
+      def self.add_notify_behavior
```

Comment:
Hmm...though I originally but these in the `DeprecationToolkit::Minitest::Plugin` namespace, I'm going to move them to the top `DeprecationToolkit` namespace. The reason is that they aren't specific to minitest, so if rspec support is going to be added (#17), it would be weird to call `DeprecationToolkit::Minitest::Plugin.*` in an rspec plugin.